### ISSUE TITLE
marked as a bot

### ISSUE THREAD

**Issue by XPeTerchenX** [CommanderRoot/twitch-tools#94]
Hello,

i`m moderator from zbrokoli https://www.twitch.tv/zbrokoli
but i (19_peterchen_97) cant`t follow her or see her content.
yesterday i see that i`m listed as a bot how long does it take too remove me from that status as abot  and what should zbrokoli do after i removed?

**Comment by CommanderRoot** [comment]
Duplicate of #30

Your account was added to the know bot list on `2021-12-01` as it was part of one or multiple follow bot raids back then. It's quite possible that your account was compromised at the time.
Your account was removed from the list yesterday evening.

To be able to follow the broadcaster has to unblock you. The easiest way they can do that that would be typing "/unblock 19_peterchen_97" in their chat
